src/app/components/hris/employees/employee-profile/accordions/accordion-career-summary/accordion-career-summary-qualifications/accordion-career-summary-qualifications.component.ts

```diff
@@ -0,0 +1,240 @@
+import { Component, HostListener, Input } from '@angular/core';
+import { EmployeeProfile } from 'src/app/models/hris/employee-profile.interface';
+import { SimpleEmployee } from 'src/app/models/hris/simple-employee-profile.interface';
+import { SharedAccordionFunctionality } from '../../../shared-accordion-functionality';
+import { SharedPropertyAccessService } from 'src/app/services/hris/shared-property-access.service';
+import { PropertyAccessLevel } from 'src/app/models/hris/constants/enums/property-access-levels.enum';
+import { AbstractControl, FormBuilder, Validators } from '@angular/forms';
+import { SnackbarService } from 'src/app/services/shared-services/snackbar-service/snackbar.service';
+import { EmployeeQualificationsService } from 'src/app/services/hris/employee/employee-qualifications.service';
+import { EmployeeQualifications } from 'src/app/models/hris/employee-qualifications.interface';
+import { NavService } from 'src/app/services/shared-services/nav-service/nav.service';
+
+
+@Component({
+  selector: 'app-career-summary-qualifications',
+  templateUrl: './accordion-career-summary-qualifications.component.html',
+  styleUrls: ['./accordion-career-summary-qualifications.component.css']
+})
+
+export class CareerSummaryQualificationsComponent {
+
+  screenWidth = window.innerWidth;
+  @HostListener('window:resize', ['$event'])
+ 
+  onResize() {
+    this.screenWidth = window.innerWidth;
+  }
+
+  constructor(
+    public sharedAccordionFunctionality: SharedAccordionFunctionality,
+    public sharedPropertyAccessService: SharedPropertyAccessService,
+    private snackBarService: SnackbarService,
+    private fb: FormBuilder,
+    private employeeQualificationsService : EmployeeQualificationsService,
+    public navservice: NavService
+  ) {
+  }
+
+  usingProfile: boolean = true;
+  isValidFile: boolean = false;
+  isValidFileSize: boolean = false;
+  fileUploaded: boolean = false;
+
+  fileName: string = '';
+  filePreview: string | ArrayBuffer | null = null;
+  base64File: string = "";
+  fileUrl: string = '';
+  proofOfQualificationFinal: string = '';
+  isDisabledUpload: boolean = true;
+  isDisabledDownload: boolean = true;
+
+  ngOnInit() {
+    this.usingProfile = this.employeeProfile!.simpleEmployee == undefined;
+    this.fetchQualificationsById();
+  }
+
+  @Input() employeeProfile!: { employeeDetails: EmployeeProfile, simpleEmployee: SimpleEmployee }
```

Comment:
Move the input property to the top of the proporties, it is a common Angular standard to declare them at the top of other properties